google-cloud-core-bom/pom.xml

```diff
@@ -75,6 +75,11 @@
         <artifactId>google-cloud-core-http</artifactId>
         <version>2.3.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-core:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core-native-support</artifactId>
+        <version>2.3.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-core:current} -->
```

Comment:
This would mark the native support as GA (post 1.0). We should define a version that matches the artifact status.

Yoshi automation tool has capability to define different versions in a repository. For example gax-httpjson is pre 1.0 but gax is 2.7.0.
https://github.com/googleapis/gax-java/blob/main/gax-httpjson/build.gradle#L4

Reply:
Done.